teal/legendary/utils.tl

```diff
@@ -304,6 +304,21 @@ function M.append_trailing_cr(cmd_str: string): string
   return cmd
 end
 
+
+--- Check if given item is a user-defined function
+---@param cmd any
+---@return boolean
+function M.is_user_function(cmd: LegendaryFunction): boolean
+  return not not (
+      cmd ~= nil
+      and type(cmd) == 'table'
+      and type(cmd[1]) == 'function'
+      and type(cmd[2]) == 'string'
```

Comment:
What are we checking for at the 2nd index here? Aren’t function tables just like:

```lua
{
  function()
    -- stuff
  end,
  description = “A function”
}
```

Reply:
oops, probably copy pasted. Fixed.